blueprints/component/glimmer-files/__root__/__path__/__name__.ts

```diff
@@ -1,5 +1,9 @@
 import Component from '@glimmer/component';
 
-interface <%= classifiedModuleName %>Args {}
+interface <%= classifiedModuleName %>Args {
+  // define component arguments here
+}
 
-export default class <%= classifiedModuleName %> extends Component<<%= classifiedModuleName %>Args> {}
+export default class <%= classifiedModuleName %> extends Component<<%= classifiedModuleName %>Args> {
```

Comment:
See #214 .

Reply:
This is pasted from the default Ember blueprints

Reply:
Yep, and we'll keep it as is, esp. for now!